src/js/HScrollMenuItem.js

```diff
@@ -1,26 +1,32 @@
 import React from 'react';
 import { Link } from 'react-router';
+import Image from './Templates/Shared/Image'
+import StaticIcon from './Templates/Shared/StaticIcon'
 
 export default class HScrollMenuItem extends React.Component {
     constructor(props) {
         super(props);
     }
     render() {
         const { menuItem } = this.props;
-        const graphic = this.props.menuItem.class == 'with-icon' ?
-        (<span className="svg-wrap" dangerouslySetInnerHTML={{__html: `${menuItem.image}`}} />)
-        : <img src={menuItem.image} /> ;
+        var graphic = (this.props.menuItem.imageType === "STATIC")
+                 ? <StaticIcon image={this.props.menuItem.image} />
+                 : <Image image={this.props.menuItem.image} />;
+
         return (
-            <div
-                className={`hscrollmenu-item th-b-color th-tile-background-color ${menuItem.class}`}
+            <Link
+                to={this.props.menuItem.link}
                 onClick={() => this.props.onSelection(this.props.appID, this.props.cmdID, this.props.menuID)}>
-                <div className="hscrollmenu-item__image">
-                    { graphic }
-                </div>
-                <div className="hscrollmenu-item__name">
-                    <p className="t-small t-light th-f-color">{menuItem.name}</p>
+                <div
+                    className={`hscrollmenu-item th-b-color th-tile-background-color`}>
```

Comment:
Since the className is not being modified dynamically could simply change it to 
```suggestion
                    className="hscrollmenu-item th-b-color th-tile-background-color">
```